runtime/src/accounts.rs

```diff
@@ -476,7 +476,7 @@ impl Accounts {
         }
     }
 
-    pub fn hash_internal_state(&self, slot_id: Slot) -> BankHash {
```

Comment:
Changed from `verb_object` to `noun_preposition` style here, as it's rather a getter `fn`, not doing actual calculating here.